packages/p2p-v2/src/hooks/usePoiPoaStatus.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import { useGetAccountStatus } from '@deriv/api';
+
+/** A custom hook that returns the POA, POI status and if POA is required for P2P */
```

Comment:
Can we write a better description for this, it's confusing for me 🤔

Reply:
hmmm which part is confusing specifically?

Reply:
Please ignore this comment @ameerul-deriv 😁